docs/eventlisteners.md

```diff
@@ -22,6 +22,7 @@ using [Event Interceptors](#Interceptors).
     - [Interceptors](#interceptors)
 - [Logging](#logging)
 - [Labels](#labels)
+- [EventListener Liabilities](#eventlistener-liabilities)
```

Comment:
Maybe we just call this section "How does the EventListener work?"

Reply:
Done